src/xdocs/config_javadoc.xml

```diff
@@ -227,13 +227,13 @@ public int checkReturnTag(final int aTagIndex,
           <tr>
             <td>scope</td>
             <td>visibility scope where Javadoc comments are checked</td>
-            <td><a href="property_types.html#scope">scope</a></td>
+            <td><a href="property_types.html#scope">Scope Policy</a></td>
```

Comment:
I can make this scope if you wish.
I think I added policy because other enumerations said policy like ["left curly brace policy"](http://checkstyle.sourceforge.net/config_blocks.html#LeftCurly) and ["block policy"](http://checkstyle.sourceforge.net/config_blocks.html#EmptyBlock) and ["closing parens"](http://checkstyle.sourceforge.net/config_annotation.html#AnnotationUseStyle) (notice type doesn't say policy but description does).